app/cluster_data.go

```diff
@@ -30,3 +30,8 @@ func addLocalCluster(embedded bool, adminName string, management *config.Managem
 	management.Management.Clusters("").Create(c)
 	return nil
 }
+
+func removeLocalCluster(management *config.ManagementContext) error {
+	management.Management.Clusters("").Delete("local", &v1.DeleteOptions{})
+	return nil
```

Comment:
why even have error as a return parameter if you swallow the error and dont ever return any?

Reply:
no reason of mine, just followed the logic of `addLocalCluster` which doesn't return the error and there's a comment `// Ignore error`